pkg/csi/cinder/openstack/openstack.go

```diff
@@ -27,6 +28,7 @@ import (
 	"github.com/spf13/pflag"
 	gcfg "gopkg.in/gcfg.v1"
 	openstack_provider "k8s.io/cloud-provider-openstack/pkg/cloudprovider/providers/openstack"
+	md "k8s.io/cloud-provider-openstack/pkg/util/metadata"
```

Comment:
why not just use `metadata` as package name?

Reply:
Change missing?

Reply:
I think I added a comment ,why it's not showing? werid..

I can't do so because if I move to metadata, it will be duplicate to openstack_metadata.go
the unit will fail